### DISCUSSION TITLE
iOS cache hit (direct) = 0

### DISCUSSION THREAD

**Discussion by namedssy** [ccache/ccache#295] [Support]
Has anyone encountered this situation please?

cache directory                     /Users/uxin_ssy/.ccache
primary config                      /Users/uxin_ssy/.ccache/ccache.conf
secondary config      (readonly)    /usr/local/Cellar/ccache/3.4.2/etc/ccache.conf
stats zero time                     Sat Aug 11 15:19:32 2018
cache hit (direct)                     0
cache hit (preprocessed)               0
cache miss                          6459
cache hit rate                      0.00 %
called for link                       16
called for preprocessing               6
compile failed                         9
preprocessor error                     1
unsupported compiler option         3954
no input file                          5
cleanups performed                     0
files in cache                     30631
cache size                           2.3 GB
max cache size                      10.0 GB

**Comment by afbjorklund** [comment]
Seems like you are getting *no* hits, whether direct or preprocessed ?

Is this a second build, where the cache is supposed to be warm ?
(the first build is always cold, so there the results are to be expected)

I guess these could also be interesting: `unsupported compiler option 3954`

**Comment by jrosdahl** [comment]
@namedssy: Please see https://github.com/ccache/ccache/blob/master/CONTRIBUTING.md#reporting-an-issue

**Comment by namedssy** [comment]
thank

**Comment by jrosdahl** [comment]
@namedssy: Closing this now, please comment if you're still wondering something.
